app/services/to_work_form/mapper.rb

```diff
@@ -56,5 +57,18 @@ def release_date_params
 
       { release_date:, release_option: 'delay' }
     end
+
+    def custom_rights_statement
+      use_statement = CocinaSupport.use_reproduction_statement_for(cocina_object:)
+      return if use_statement.blank?
+      return if use_statement == default_terms_of_use
+
+      # Remove the default terms
+      use_statement&.delete_suffix(default_terms_of_use)&.strip
```

Comment:
We're short-circuiting a few lines above if the use statement is blank, so I'm curious about the safe navigation here.

Reply:
Yeah, that got left behind when I refactored this method. Removing.